.github/workflows/tag.yml

```diff
@@ -43,9 +43,18 @@ jobs:
           current_version: ${{ steps.get-latest-tag.outputs.tag }}
           level: ${{ steps.release-label.outputs.level }}
 
+      - name: Get commit ID for the new tag
```

Comment:
I'm not in a situation right now to look it up, but there's no variable, for say something like this `github.event.pull_request.commit_id`?

Reply:
Because that didn't work. That's the whole point of this PR to fix this

Reply:
Ah, my bad. What about this environment variable instead:
```
steps.pr.outputs.number
```
We have the `actions-ecosystem/action-get-merged-pull-request@v1.0.1` workflow, which should already do this for us

Reply:
Okay, my bad. Then this should be fine